config/admins.txt

```diff
@@ -135,3 +135,4 @@ Naksuasdf = Game Master
 MrDoomBringer = Game Master
 shizcalev = Game Master
 NicBR = Game Master
+LoserWasTaken = Game Master

```

Comment:
lol

Reply:
he tried

Reply:
it was for testing and it doesn't matter anyway the servers use sql